pkg/reconciler/v1alpha1/taskrun/resources/pod.go

```diff
@@ -90,8 +90,8 @@ const (
 	// Prefixes to add to the name of the init containers.
 	// IMPORTANT: Changing these values without changing fluentd collection configuration
 	// will break log collection for init containers.
```

Comment:
i think you can remove this comment and/or update it! looks terribly out of date - I think it's left over from knative/build (fluentd is definitely not involved in anything we're currently doing)

Reply:
done :)